draft.md

```diff
@@ -111,7 +111,7 @@ A natural language approach to smart contracts will attract the millions of indi
 
 ## 4.2 Oracles
 
-Smart contract creators place oracles as pre-built components into their ‘Lego box’. Developers are incentivized to create new oracles through a pay-per-transaction model (i.e., each time an oracle provides an answer, the developer of the oracle is paid). We aim to gather a library of oracle components to satisfy an ever-broadening scope for smart contracts, similar to how the Unity 3D Asset Store provides a database of 3D characters for game developers.
```

Comment:
Please leve the single quotes in place everywhere. These may be important when processing the whitepaper. Please try to focus on substantial changes and limit the tweaking.

Reply:
Done!